src/chrome/content/rules/Here.com.xml

```diff
@@ -40,48 +22,61 @@
 		- stg.static.here.com
 		- stg.here.com
 		- www.here.com
-
 -->
 <ruleset name="Here.com (partial)">
```

Comment:
I think it's safe to remove the partial flag here, because most hosts are supported.